src/xcode/ENA/ENA/Source/Scenes/Events/TraceLocations/Info/TraceLocationsInfoViewController.swift

```diff
@@ -34,10 +34,21 @@ class TraceLocationsInfoViewController: DynamicTableViewController, FooterViewHa
 				self?.onDismiss(false)
 			})
 		}
+		navigationController?.presentationController?.delegate = self
 		navigationItem.title = AppStrings.TraceLocations.Information.title
 		navigationController?.navigationBar.prefersLargeTitles = true
 	}
 	
+	// MARK: - Protocol UIAdaptivePresentationControllerDelegate
+	
+	func presentationControllerShouldDismiss(_ presentationController: UIPresentationController) -> Bool {
+		return false
+	}
+
+	func presentationControllerDidAttemptToDismiss(_ presentationController: UIPresentationController) {
+		self.onDismiss(false)
+	}
+	
```

Comment:
That's pretty fine. 
But to avoid a growing list of implementing protocols and Mark comments, I would prefer an extension

```swift 
extension TraceLocationsInfoViewController: UIAdaptivePresentationControllerDelegate { ... }
```

Reply:
Thanks @flxschmidt, I have implemented the extension accordingly.